src/Publication/PublicationRequestFilters.js

```diff
@@ -26,53 +27,27 @@ export class PublicationRequestFilters {
   }
 
   sanitize(req) {
+    let by = server.extractFromReq(this.req, 'sort_by')
+    let order = server.extractFromReq(this.req, 'sort_order')
+    let limit = server.extractFromReq(this.req, 'limit')
+    let offset = server.extractFromReq(this.req, 'offset')
+
+    by = by in ALLOWED_VALUES ? by : DEFAULT_VALUES['sort_by']
+    order = ALLOWED_VALUES[by].includes(order.toUpperCase())
+      ? order
+      : ALLOWED_VALUES[by][0]
+    limit = Math.max(Math.min(100, limit), 0)
+    offset = Math.max(offset, 0)
```

Comment:
Maybe this 0 and the above 100 should be obtain from `DEFAULT_VALUES` ? or just delete `DEFAULT_VALUES` completely, no?

Reply:
I removed DEFAULT_VALUES